python/thrift/provision.sh

```diff
@@ -13,7 +13,7 @@ sudo apt-get install -y build-essential libboost-dev libboost-test-dev libboost-
 # The Debian instructions on the Thrift website recommend this automake build
 # https://thrift.apache.org/docs/install/debian
 cd /tmp
-wget http://ftp.debian.org/debian/pool/main/a/automake-1.14/automake_1.14.1-4_all.deb
+wget http://ftp.debian.org/debian/pool/main/a/automake-1.14/automake_1.14.1-3_all.deb
```

Comment:
It's at http://ftp.debian.org/debian/pool/main/a/automake-1.14/automake_1.14.1-4_all.deb.

I think the instructions below are wrong.